runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -181,6 +182,65 @@ public static void renderHoverableArea(Graphics2D graphics, Area area, net.runel
 		}
 	}
 
+	public static void renderNPCList(Graphics2D graphics, Client client, List<NPC> actorList, Color color, OverlayStyle style, boolean drawNames)
```

Comment:
Why this accepts list again? I mentioned that on previous PR, this shoud just take Actor as parameter